app/workflows/lists/conversion/involuntary/sections/kickoff.yml

```diff
@@ -40,7 +40,8 @@ tasks:
     title: External stakeholder kick-off
     hint:
       You should start this work after you've had the handover with the regional
-      delivery officer.
+      delivery officer. This is a good opportunity for you to talk to about
```

Comment:
Is there an extra "to" in this sentence that shouldn't be there? I've crossed out the one that I feel is there in error.

> This is a good opportunity for you to talk ~~to~~ about common problems and how to avoid them.

or is a "them" needed to be added in, like:

> This is a good opportunity for you to talk to them about common problems and how to avoid them.

Reply:
A nice catch, I will remove the extra to as I think having 'This is a good opportunity to talk to them about…' is adding happy words that do not add to the understanding of the sentence.